main/wiki/wiki.inc.php

```diff
@@ -736,6 +737,7 @@ public function save_new_wiki($values)
                     ->setEditlock(0)
                     ->setIsEditing(0)
                     ->setTag('')
+                    ->setPageId(intval($_clean['page_id']))
```

Comment:
I don't understand why you set the page_id here when the page id will be created once the new wiki is created and then we will assign the page_id base on the id return by this call. This page_id assignment will happen on line 760.

So I would not leave what you put on lines 715 and 749